ref/general/os-hardening.adoc

```diff
@@ -0,0 +1,86 @@
+Using a minimal image reduces the number of OS vulnerabilities that you bundle with your application.
+
+=== Build and secure an Open Liberty Docker image
+If you want to build your own Open Liberty Docker image, you need to secure and harden the image before you use it in production.
+
+// Confirm the following things about your Open Liberty image before it runs in production:
+
+// * ONE
+
+== Stay current with Open Liberty
+Keep your version of Open Liberty up to date.
+Open Liberty's zero migration policy means that you can move to a later version without changing anything in your server configuration.
+Staying on the latest version also ensures that you get the latest link:/docs/ref/general/#security-vulnerabilities.html[security fixes] because updates are first released in the current version.
+After security fixes are released for the current version of Open Liberty, they are rolled back to older releases.
+
+== Validate installation
+You can validate your Open Liberty server installation by running the `/<wlp_install_dir/bin/productInfo validate` command.
+This command uses an MD5 checksum to check your installation for errors or potential vulnerabilities.
+The output message that you receive indicates whether Open Liberty was validated successfully.
+
+== UNIX file permissions
+A critical aspect of OS security is ensuring that users have the correct permissions.
+This means that users must fall into the appropriate UNIX permissions bit.
+UNIX file permissions are calculated with three bits that represent read, write, and execute access.
+The base-2 values are converted to decimal values that indicate the file permissions of a user.
+`7` indicates a user has *owner* file permissions, `5` indicates a user has *group* file permissions, and `0` indicates a user has *other* file permissions, meaning they are not part of the group.
+By default, Open Liberty Docker images run with the `USER 1001` non-root user as part of group `0`.
+The following diagram depicts the UNIX file permissions:
+
+<diagram pending>
+
+== Access control
+It is critical to control users' access levels so that all users fall into the appropriate UNIX file permissions bits.
+Limit users to have minimal required access based on their needs.
+Different types of users might try to access the configuration:
+
+* *The ID that the Open Liberty server uses to run*: This ID needs to read its configuration and write to logs.
+This ID can also own files, or it can be a separate ID from the file owner.
+* *Users who are responsible for administering the server*: These users need read and write access.
+Don't share login IDs or passwords among administrators.
+* *Users who are involved in server-related activities*: Some of these users need read access to view the logs, while others might need limited write access.
+* *Unauthorized users*: These users have no access at all and fall into the other permissions bit that is indicated by a `0` UNIX file permissions.
```

Comment:
Change,
"*Unauthorized users*: These users have no access at all and fall into the other permissions bit that is indicated by a `0` UNIX file permissions."
to,
"*Unauthorized users*: These users have no access and fall into the other permissions bit that is indicated by a `0` UNIX file permissions."